packages/fast-usdc/test/fast-usdc.contract.test.ts

```diff
@@ -213,12 +219,32 @@ test('LP deposits, earns fees, withdraws', async t => {
 
   await Promise.all([lps.alice.deposit(60n), lps.bob.deposit(40n)]);
   {
-    const feeAmt = usdc.make(25n);
-    t.log('contract accrues some amount of fees:', ...logAmt(feeAmt));
-    const feePmt = await utils.pourPayment(feeAmt);
-    await E(creatorFacet).simulateFeesFromAdvance(feeAmt, feePmt);
+    t.log('simulate borrow and repay so pool accrues fees');
+    const feeTools = makeFeeTools(makeTestFeeConfig(usdc));
+    const requestedAmount = usdc.make(100n);
+    const splits = feeTools.calculateSplit(requestedAmount);
+
+    const amt = await E(creatorFacet).testBorrow({ USDC: splits.Principal });
+    t.deepEqual(
+      amt.USDC,
+      splits.Principal,
+      'testBorrow returns requested amount',
+    );
+
+    const repayPayments = {};
+    for (const [kw, amount] of Object.entries(splits)) {
+      const pmt = await utils.pourPayment(amount);
+      repayPayments[kw] = pmt;
+    }
```

Comment:
This should avoid the `@ts-expect-error` below:

```suggestion
    const repayPayments = await deeplyFulfilledObject(objectMap(splits, utils.pourPayment));
```